continuous_integration/install.sh

```diff
@@ -76,3 +76,4 @@ if [ -z "$NUMPY_VERSION" ]; then
 fi
 
 python setup.py install
+pip install nose-exclude
```

Comment:
Do you really need this. I played a little bit with nose-exclude and the impression I got was that it was to exclude tests not to remove files from the coverage which is probably what we would want here.